### PR TITLE
Some notes I took while trying to review the animation system.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @abarth. I don't know how useful this really is.

File: animations.md [hunk 1]
```diff
@@ -112,3 +114,198 @@ with an animation to get a concrete value:
...
+elapsed since it was started and passes it to a `Simulation` to obtain
+a value. That is then the value it reports. If the `Simulation`
+reports that at that time it has ended, then the controller stops
+itself.
+
+The animation controller can be given a lower and upper bound to
+animate between, and a duration.
+
+In the simple case (using `forward()`, `reverse()`, `play()`, or
+`resume()`,), the animation controller simply does a linear
```
Comment (abarth) [inline animations.md]: extra comma here.

File: animations.md [hunk 2]
```diff
@@ -112,3 +114,198 @@ with an animation to get a concrete value:
...
+interpolation over the given duration from the current value to the
+given target. If no duration is given to the method, the default
+duration of the controller and the range described by the controller's
+lower bound and upper bound is used to determine the velocity of the
+animation.
+
+When using `fling()`, a `Force` is used to create a specific
+simulation which is then used to drive the controller.
+
+When using 'animateWith()`, the given simulation is used to drive the
```
Comment (abarth) [inline animations.md]: Missing a backtick

**Comment by abarth** [comment]
LGTM

**Comment by krisgiesing** [comment]
I like it :)

**Pull request merged**